docs/install/collecting-metrics/collector.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+  namespace: 
+data:
+  collector.yaml: |
+    receivers:
+      opencensus:
+        endpoint: "0.0.0.0:55678"
+
+    exporters:
+      logging:
+      prometheus:
+        endpoint: "0.0.0.0:8889"
+
+    service:
+      extensions:
+        health_check:
+        pprof:
+        zpages:
```

Comment:
Do these extensions work if you don't declare them first?  Something like this is what I was expecting: https://github.com/open-telemetry/opentelemetry-collector/blob/master/examples/demo/otel-collector-config.yaml#L30

If these aren't functional, should we take them out?

Reply:
These have defaults assuming the key is present. No arguments are needed (I added a port declaration for the `zpages` port; I can add the other two port declarations to the Deployment if you think it helps).

Reply:
If they will work this way, I'd say leave them in.  I've always declared them above, so wasn't sure.

Also, don't think it's necessary to declare all of the ports.  ZPages is the main one, so.